src/Evolution/DiscontinuousGalerkin/NormalVectorTags.hpp

```diff
@@ -0,0 +1,41 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <optional>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Domain/Structure/Direction.hpp"
+#include "Domain/Structure/DirectionMap.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// Tags used on the interior faces of the elements
+namespace evolution::dg::Tags::InternalFace {
```

Comment:
can't find the namespace `InternalFace` in the documentation (can't find `Tags` either).   Are these tags only needed on internal faces?

Reply:
I have not yet decided if they are only for the internal faces. Currently they are. However, currently it looks like they will be used for external faces too, though I'd prefer to defer the rename until I know for sure that's what I need them for. IS that okay?

Reply:
yeah that's fine

Reply:
I think the rename would be `Internal` -> `Interior`? I can't remember the bloody names -_-

Reply:
So my take on our terminology is as follows:  each Element has a set of faces that are either internal or external depending on whether or not they have any neighboring Elements.  Between an Element and each of its neighbors there is a Mortar (which is between exactly two Elements).  Since we duplicate the work for the mortars in each Element, then when referring to a Mortar for a given Element, we refer to data from the Element as interior data, and data from the neighboring Element as exterior data.  And in some cases on an external face of an Element, an external boundary condition can be imposed using a Mortar with the exterior data specified by the concrete boundary condition class.    If interior and exterior are confusing maybe they should be renamed to (element/my/self) data and neighbor data

Reply:
Yea, that's my understanding. If I use them for the external and internal interior faces I'll rename the namespace (and update comments) to `InteriorFaces`.